src/main/java/uk/gov/laa/ccms/caab/client/CaabApiClient.java

```diff
@@ -33,7 +33,58 @@ public class CaabApiClient {
 
   private final WebClient caabApiWebClient;
 
-  private final CaabApiClientErrorHandler caabApiClientErrorHandler;
+  private final ApiClientErrorHandler apiClientErrorHandler;
+
+  public static final String FAILED_CREATE_APPLICATION =
```

Comment:
I think maybe you were right to suggest refactoring further to remove the need for all these constants. As per my slack msg.

Reply:
amended